pythonforandroid/recipes/sdl2_mixer/toggle_modplug_mikmod_smpeg_ogg.patch

```diff
@@ -1,29 +1,28 @@
---- orig/Android.mk	2016-01-03 07:15:57.000000000 +0100
-+++ patch/Android.mk	2016-04-15 21:28:55.169697882 +0200
-@@ -6,22 +6,22 @@
- 
- # Enable this if you want to support loading MOD music via modplug
- # The library path should be a relative path to this directory.
--SUPPORT_MOD_MODPLUG ?= true
-+SUPPORT_MOD_MODPLUG := false
- MODPLUG_LIBRARY_PATH := external/libmodplug-0.8.8.4
- 
- # Enable this if you want to support loading MOD music via mikmod
- # The library path should be a relative path to this directory.
--SUPPORT_MOD_MIKMOD ?= true
-+SUPPORT_MOD_MIKMOD := false
- MIKMOD_LIBRARY_PATH := external/libmikmod-3.1.12
- 
- # Enable this if you want to support loading MP3 music via SMPEG
- # The library path should be a relative path to this directory.
--SUPPORT_MP3_SMPEG ?= true
-+SUPPORT_MP3_SMPEG := false
- SMPEG_LIBRARY_PATH := external/smpeg2-2.0.0
- 
+diff -Naur SDL2_mixer/Android.mk SDL2_mixer-1/Android.mk
+--- SDL2_mixer/Android.mk	2019-03-07 21:07:42.149457400 +0100
++++ SDL2_mixer-1/Android.mk	2019-03-07 21:05:06.851338800 +0100
+@@ -3,20 +3,20 @@
+
+
+ # Enable this if you want to support loading FLAC music with libFLAC
+-SUPPORT_FLAC := false
++SUPPORT_FLAC ?= true
+ FLAC_LIBRARY_PATH := external/flac-1.3.2
+
  # Enable this if you want to support loading OGG Vorbis music via Tremor
- # The library path should be a relative path to this directory.
--SUPPORT_OGG ?= true
-+SUPPORT_OGG := true
- OGG_LIBRARY_PATH := external/libogg-1.3.1
+-SUPPORT_OGG := true
++SUPPORT_OGG ?= true
+ OGG_LIBRARY_PATH := external/libogg-1.3.2
  VORBIS_LIBRARY_PATH := external/libvorbisidec-1.2.1
- 
+
+ # Enable this if you want to support loading MP3 music via MPG123
+-SUPPORT_MP3_MPG123 := false
++SUPPORT_MP3_MPG123 ?= true
```

Comment:
this library is LGPL. I don't think it should be included per default, I think that's a problematic change

Reply:
@misl6 can you explain why you marked this conversation as resolved? Unless it's a fairly obvious case (in which it would be still somewhat expected you spell it out why you resolved it), please don't resolve things brought up by others because people may then forget to keep track of them